sample-data/lessons.csv

```diff
@@ -1212,19 +1232,139 @@ Id,Lesson,Slug,Description,Excerpt,Status,Module,Prerequisite,Preview,Tags,Image
 <p>Drilling down into a particular lesson displays a page that lists all of the learners who are currently taking or have already completed the lesson. The view is the same as <a href=""#course-lesson-learners"">this one</a>.</p>
 <!-- /wp:paragraph -->","Learn how to track your learners' course progress in Sensei LMS.",publish,,,0,,,,,,0,0,,0,0,0,0,
 3175,Questions,questions,"<!-- wp:paragraph -->
-<p>Questions can be added independently of a quiz, making it possible to use them across multiple quizzes.</p>
+<p>Questions can be added in one of two ways - by going to <em>Questions</em> &gt; <em>Add New</em>, or by adding them directly to a quiz. Although this lesson covers the latter, configuring questions is the same regardless of which method is used.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>To add questions to a quiz, create a new lesson by going to <em>Lessons</em> &gt; <em>Add New</em>, or select an existing one from <em>Lessons</em> &gt; <em>All Lessons</em>. For a new lesson, you will see that the quiz block has already been added to the page for you. For an existing lesson, you can add the quiz block by searching for it in the <a href=""https://wordpress.org/support/article/adding-a-new-block/#what-is-the-inserter"">block inserter</a>. <em>(Note: For managing questions using the legacy meta box rather than blocks, please refer to <a href=""#legacy"">this documentation</a>.)</em></p>
```

Comment:
Not sure if this is inaccurate, but just in case, this is the behavior: 
- Quiz blocks are automatically added to the lesson if a quiz has been added to the lesson (either initially through block editor or metabox editor). 
- Once automatically or manually added, removing a quiz block deactivates the quiz from the lesson. Adding the quiz block back will reactivate the quiz and restore previous questions.
- If a lesson doesn't have a quiz (if it didn't have any questions in the legacy editor or if the block was manually removed), they can add it back.

Reply:
I added some blurbs for removing a quiz and the conditions in which the legacy meta boxes are used. I'm intentionally not highlighting the meta boxes much though, as the blocks should be the default experience for most new users. The plan is to reinforce these points more in the release post.

Reply:
Cool. I was more wondering about the mentioning of adding a quiz block

Reply:
Ah, ok. I removed the entire bit about adding a quiz to existing lessons in https://github.com/Automattic/sensei/pull/4083/commits/7f502cd6c9a68f36f143ddf2688d45da135fc3d2 (and also picked up some encoding things 🤷‍♀️ ).